package.json

```diff
@@ -40,6 +43,10 @@
     "sinon": "^2.1.0",
     "sinon-chai": "^2.9.0",
     "style-loader": "^0.14.1",
-    "webpack": "^2.2.1"
+    "webpack": "^2.2.1",
+    "webpack-node-externals": "^1.5.4"
+  },
+  "dependencies": {
+    "hpq": "^1.0.1"
```

Comment:
I agree, it's a handy tool. I just thought maybe it could be one of the root `modules` here but you're right. It could be handy outside the editor and WordPress. Let's keep it this way.